test-locally.sh

```diff
@@ -0,0 +1,93 @@
+# This script is meant to help the local development of the core design system by building child & applications to test local changes
+
+# PRE-REQUISTES
+# 1. This script assumes that this project, both child systems, and any application project all share a parent directory on your machine.
+#    For example, your directory structure should be similar to
+#       projects/
+#           design-system/
+#           mgov-design-system/
+#           hcgov-design-system/
+#           application-to-test/
+
+# 2. Ensure that child design system projects are looking at local version of the core DS by updating the package.json so that the following dependencies look like this:
+#   "@cmsgov/design-system": "file:../design-system/packages/design-system",
+#   "@cmsgov/design-system-docs": "file:../design-system/packages/design-system-docs",
+#   "@cmsgov/design-system-scripts": "file:../design-system/packages/design-system-scripts",
+
+# 3. Ensure that application project is pointing at local version of child design system by modifying the package.json so that the following dependencies look like this:
+#   "@cmsgov/ds-healthcare-gov": "file:../hcgov-design-system",
+#       or
+#   "@cmsgov/ds-medicare-gov": "file:../mgov-design-system",
+
+# USING THIS SCRIPT
+# In the root directory of the design-system project, run `./test-locally.sh`
+#   This will install dependencies and run the core DS.
+#   Then, it will navigate to the child DS project. It will remove node_modules, install dependencies & build child system.
+#   Then, it will navigate to an application project. It will remove node_modules, install dependencies & build the application.
+
+# This script will try to use the default variables defined in this file, but you can override these for additional testing.
+# For example, if you want to test the hcgov child system, when the default CHILD_DS_NAME variable is mgov child system, you can:
+#   1. Update the CHILD_DS_NAME variable in this file
+#   2. Run `./test-locally.sh -c hcgov-design-system
+
+# Similarly, if you want to change the application path your are testing with, you can:
+#   1. Update the APP_NAME path variable in this file
+#   2. Run `./test-locally.sh -a [directory name of application]
+#     a. If you only want to test in child systems and not in an application, you can run `./test-locally.sh -a ""`
+
+#!/bin/sh
+
+set -e
```

Comment:
When I run `./test-locally.sh`, nothing happens unless I comment out this line 40

Reply:
🤔 interesting... I copied this from the prerelease script, so I'll double check what it does and remove appropriately